packages/@glimmer/wire-format/index.ts

```diff
@@ -15,7 +15,8 @@ export function isAttribute(val: Statement): val is Statements.Attribute {
     val[0] === SexpOpcodes.StaticAttr ||
     val[0] === SexpOpcodes.DynamicAttr ||
     val[0] === SexpOpcodes.TrustingDynamicAttr ||
-    val[0] === SexpOpcodes.ComponentAttr
+    val[0] === SexpOpcodes.ComponentAttr ||
+    val[0] === SexpOpcodes.Modifier
```

Comment:
This list need to be updated: https://github.com/glimmerjs/glimmer-vm/blob/master/packages/@glimmer/interfaces/lib/compile/wire-format.d.ts#L209-L214, otherwise it will cause a type error when you try to push it.

Reply:
By the way, I think you should just include the check for `AttrSplat` in here and and delete the `isAttrSplat` function. I don't know why it is handled separately, and as far as I can tell, it doesn't have to be, and probably shouldn't.

Reply:
Done!

Reply:
@cibernox still need to update the wire-format types 😄

Reply:
I just pushed that